python/ray/tune/experiment/trial.py

```diff
@@ -811,7 +811,7 @@ def has_reported_at_least_once(self) -> bool:
 
     @property
     def node_ip(self):
-        return self.location.hostname
```

Comment:
Should we add a simple unit test for verifying the attributes exist?

Reply:
oh oops I merged this before adding a test -- can do in a followup pr